gateway/src/args.rs

```diff
@@ -29,6 +35,9 @@ pub struct StartArgs {
     /// Address to bind the user plane to
     #[arg(long, default_value = "127.0.0.1:8000")]
     pub user: SocketAddr,
+    /// Allows to disable the use of TLS in proxying deployed services (DANGEROUS)
+    #[arg(long, default_value = "enable")]
+    pub use_tls: UseTls,
```

Comment:
Wouldn't a `bool` have done the same?

Reply:
It is more difficult to parameterise an optional flag with docker-compose command arguments than it is to parameterise an option.